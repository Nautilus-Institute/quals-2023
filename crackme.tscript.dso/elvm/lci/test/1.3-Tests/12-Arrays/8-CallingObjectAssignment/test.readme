This test checks that the calling object reference variable can be used to
assign values to variables.
