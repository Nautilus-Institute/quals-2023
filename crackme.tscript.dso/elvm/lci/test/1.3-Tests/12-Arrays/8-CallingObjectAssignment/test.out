4.56
123
890
890
